src/core/pretty.ml

```diff
@@ -227,6 +227,8 @@ let pp_command : p_command pp = fun oc cmd ->
         | Assoc_right -> " right"
       in
       out "set infix%s %f \"%s\" ≔ %a" a p s pp_qident qid
+  | P_set(P_config_unif_rule(ur))   ->
+      out "set unif_rule %a" (pp_p_rule true) ur
```

Comment:
You should use a specific function for printing unif_rule's. Otherwise it will print some \#equiv's and \#comma's...

Reply:
Perhaps we can more simply modify `pp_p_ident` to handle the "#..." identifiers

Reply:
It looks very hacky. As you have a specific parsing rule, it is not surprising to have a specific pp rule.